.github/workflows/continuous_integration.yml

```diff
@@ -59,16 +59,38 @@ jobs:
       if: always()
 
     - name: tangelo install
+      env:
+        NO_PYSCF: 1
```

Comment:
Note: the NO_PYSCF install is temporary we will soon make it an optional dependency